site/content/en/docs/faq/_index.md

```diff
@@ -23,3 +23,9 @@ Alternatively, configure `sudo` to never prompt for the commands issued by minik
 minikube's bootstrapper, [Kubeadm] (https://github.com/kubernetes/kubeadm) verifies a list of features on the host system before installing Kubernetes. in case you get this error, and you still want to try minikube anyways despite your system's limitation you can skip the verification by starting minikube with this extra option:
 
 `minikube start --extra-config kubeadm.ignore-preflight-errors=SystemVerification`
+
+## what is the resource allocation for Knative Setup using minikube?
```

Comment:
Is there any upstream link for this information ? Why does it need to mention Knative specifically

Couldn't find anything on https://knative.dev/docs/ - I guess most of it depends on the apps deployed

Reply:
it's bare minimum requirement for Knative setup using minikube to avoid apiserver flakes.
upstream link: https://github.com/knative/serving/blob/master/DEVELOPMENT.md#resource-allocation-for-kubernetes